eskipfile/remote.go

```diff
@@ -42,14 +49,19 @@ func RemoteWatch(o *RemoteWatchOptions) (routing.DataClient, error) {
 		return nil, err
 	}
 
+	t := 0
+	if o.Threshold > 0 {
+		t = o.Threshold
+	}
+
 	dataClient := &remoteEskipFile{
-		remotePath: o.remoteFile,
+		remotePath: o.RemoteFile,
 		localPath:  tempFilename.Name(),
-		threshold:  o.threshold,
-		verbose:    o.verbose,
+		threshold:  t,
```

Comment:
Why not just `o.Threshold`?
It defaults to 0 and here we do not need to be that safe in case of negative numbers or do you expect someone to set -1?

Reply:
I was doing it just as a validation, but happy to remove it if you think it's not needed.